src/components/TestnetInfo/TestnetInfo.component.tsx

```diff
@@ -1,11 +1,11 @@
 import { Typography } from '@mui/material';
 import Box from '@mui/material/Box';
 import { useSelector } from 'react-redux';
-import { isOnTestnetSelector } from '../../store/app/app.selectors';
+import { testnetMainnetSelector } from '../../store/app/app.selectors';
 
 export const TestnetInfo = () => {
-  const isOnTestnet = useSelector(isOnTestnetSelector);
-  if (!isOnTestnet) return null;
+  const testnetMainnetFlag = useSelector(testnetMainnetSelector);
+  if (testnetMainnetFlag === 'mainnet') return null;
```

Comment:
maybe we can put mainnet and testnet as some consts, what do you think?

Reply:
It's an union, so ts will prompt it anyway 
![image](https://user-images.githubusercontent.com/38423054/165517447-5acfdafa-0d10-4276-bafb-81e79d707284.png)